system/security/LdapSecurity/ldapsecurity.cpp

```diff
@@ -667,22 +686,41 @@ bool CLdapSecManager::authenticate(ISecUser* user)
     }
 
     //Verify provided signature if present
```

Comment:
Please can you add a comment clarifying how the signature differs from the password, and when each are used.